.travis.yml

```diff
@@ -28,4 +28,5 @@ before_install:
   - pip install -U --user pygments
 
 script:
-  - latexmk --halt-on-error simpler-wick.tex
+  - latexmk --halt-on-error simpler-wick-doc.tex
+  - pdftex --halt-on-error simpler-wick-plain.tex
```

Comment:
Overlays rely on driver-specific `\special` instructions.  It doesn't work with `tex` because of
```
! Undefined control sequence.
l.1 \pgfsyspdfmark
                   {pgfid1}{16350803}{48902537}
```
No idea, why the driver is not properly selected.